src/porepy/geometry/geometry_property_checks.py

```diff
@@ -58,40 +59,68 @@ def is_ccw_polygon(poly: np.ndarray) -> bool:
+    """ Checks if a polyline of three points goes in a counterclockwise direction.
+
+    The line segment going from p1 to p2 is tested vs. a third point to determine
+    whether the combined line segments (polyline) is part of a counterclockwise circle.
+    The function can test both one and several points vs. the same line segment.
 
-    The test is positive if p3 lies to left of the line running through p1 and
-    p2.
+    The test is positive if the test point lies to left of the line running from p1
+    to p2.
 
-    The function is intended for 2D points; higher-dimensional coordinates will
-    be ignored.
+    The function is intended for 2D points; higher-dimensional coordinates will be
+    ignored.
 
     Extensions to lines with more than three points should be straightforward,
     the input points should be merged into a 2d array.
 
-    Parameters:
-        p1 (np.ndarray, length 2): Point on dividing line
-        p2 (np.ndarray, length 2): Point on dividing line
-        p3 (np.ndarray): Points to be tested
-        tol (double, optional): Tolerance used in the comparison, can be used
-            to account for rounding errors. Defaults to zero.
+    Examples:
+        >>> p1 = np.array([0, 0])
+        >>> p2 = np.array([1, 1])
+        >>> p3 = np.array([[0.5, 0.3, 0.5], [0.2, 0.7, 0.1]])
+        >>> is_ccw_polyline(p1, p2, p3)
+        [False True False]
+
+        >>> p1 = np.array([0, 0])
+        >>> p2 = np.array([1, 1])
+        >>> p3 = np.array([0.5, 0.3])
+        >>> is_ccw_polyline(p1, p2, p3)
+        False
 
-        default (boolean, optional): Mode returned if the point is within the
-            tolerance. Should be set according to what is desired behavior of
-            the function (will vary with application). Defaults to False.
+    See also:
+        :func:`is_ccw_polygon`
+
+    Parameters:
+        p1 (shape=(2,)): First point on dividing line.
+        p2 (shape=(2,)): Second point on dividing line.
+        p3 (shape=(2,) or shape=(2, n)): Point(s) to be tested. For one point, only
+            arrays of shape=(2,) is accepted. For two or more points the array will
+            have shape=(2, n), where the first row corresponds to x-coordinates and
+            the second row corresponds to y-coordinates. See examples.
+        tol (optional): Tolerance used in the comparison, can be used to
+            account for rounding errors. Defaults to zero.
+
+        default: Mode returned if the point is within the tolerance. Should be set
+            according to what is desired behavior of the function (will vary with
+            application). Defaults to False.
 
     Returns:
-        boolean or np.array of booleans. true if the points form a ccw polyline.
-            An array is returned if more than one point is tested.
+        True for each test point that will form a ccw polyline with the two
+        comparing points.
 
-    See also:
-        is_ccw_polygon
+    TODO:
```

Comment:
Probably, this todo is directed towards code development and should not appear in the doc-string of the method. I would rather expect such a todo in the inner body of the code, just after the end of the doc-string.